src/components/molecules/content/portable-text/Table.svelte

```
@@ -0,0 +1,46 @@
+	<table class="min-w-full rounded-xl border border-gray-200 shadow-sm">
+		<thead class="bg-gray-100">
+			{#if rows && rows.length > 0}
+				<tr>
+					{#each rows[0].cells as cell}
+						<th
+							scope="col"
+							class="border-b border-gray-200 px-4 py-2 text-left text-sm font-semibold"
+						>
+							{cell}
+						</th>
+					{/each}
```

ESLint

[error] Each block should have a key

(svelte/require-each-key)